subprojects/core/src/main/java/org/gradle/testfixtures/internal/InMemoryIndexedCache.java

```diff
@@ -28,13 +27,13 @@
 import java.util.Map;
 import java.util.Set;
 import java.util.concurrent.ConcurrentHashMap;
-import java.util.function.Supplier;
+import java.util.function.Function;
 
 /**
  * A simple in-memory cache, used by the testing fixtures.
  */
 public class InMemoryIndexedCache<K, V> implements PersistentIndexedCache<K, V> {
```

Comment:
Not about this PR, but an `InMemoryIndexedCache` should not implement `PersistentIndexedCache`, as it is not _persistent._ We need to straighten this out eventually.

Reply:
It should, since it is an in-memory persistent cache implementation used for testing: 
> A simple in-memory cache, used by the testing fixtures.

Reply:
Should it be called something like `TestInMemoryPersistentIndexedCache` then?

Reply:
Yes. Renamed in https://github.com/gradle/gradle/pull/15067.